### DISCUSSION TITLE
Add Quick Access on File Explorer

### DISCUSSION THREAD

**Discussion by Mamukata** [valinet/ExplorerPatcher#2281] [General]
There is a way to add again Quick Access option in the left side of "File Explorer"?

Tnx

**Comment by pyrates999** [comment]
was it showing before?

**Reply by Mamukata** [reply]
I'm not sure, even if I think yes.
In any case not now.
I see the Onedrive link, Dropbox link but not the Quick access.

![image](https://github.com/valinet/ExplorerPatcher/assets/147610373/9245b15c-6cd8-400b-a0af-9c2838d2aa3f)



Regards

**Comment by pyrates999** [comment]
Quick Access is shown above Desktop in its own section.

**Comment by Mamukata** [comment]
> Quick Access is shown above Desktop in its own section.

As you can see from my screen, above Desktop I don't have any other...

Another screen with Desktop minimized:

![image](https://github.com/valinet/ExplorerPatcher/assets/147610373/b7e49432-5fe9-4279-ac0d-87d9ed060287)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
